packages/cli/src/build/config.ts

```diff
@@ -198,14 +220,19 @@ export function getRollupConfigs(pkg: Package, aliases: Aliases) {
           interop,
           plugins: cjsPlugins,
         },
-        hasModuleField && {
-          format: "es" as const,
-          entryFileNames: `[name].${getDistExtension("browser-esm")}`,
-          chunkFileNames: `dist/[name]-[hash].${getDistExtension(
-            "browser-esm"
-          )}`,
-          dir: pkg.directory,
-        },
+        ...(hasModuleField
+          ? forkDevProdConditions(
```

Comment:
I feel like it's not bad to "fork" those things here like that - for the browser and worker bundles. Both prod and dev bundles are colocated etc. The node thing though... that's a little bit of a dance. It took me a while to understand how things around it works. It's just a thought but maybe this could be refactored somehow in the future.